lib/pages/chat_details/participant_list_item.dart

```diff
@@ -29,13 +35,20 @@ class ParticipantListItem extends StatelessWidget {
     return Opacity(
       opacity: user.membership == Membership.join ? 1 : 0.5,
       child: ListTile(
-        onTap: () => showAdaptiveBottomSheet(
-          context: context,
-          builder: (c) => UserBottomSheet(
-            user: user,
-            outerContext: context,
-          ),
-        ),
+        onTap: () async {
+          final result = await showAdaptiveBottomSheet(
+            context: context,
+            builder: (c) => UserBottomSheet(
+              user: user,
+              outerContext: context,
+            ),
+          );
+          if (result is ChatMembersStatus) {
```

Comment:
unnecessary IMO 
`result == ChatMembersStatus.updated)` is enough

Reply:
it's okay, maybe we need to handle other cases of `ChatMembersStatus`